.travis.yml

```diff
@@ -1,17 +1,49 @@
-dist: "xenial"
-
 language: python
-script: python setup.py test
 
-matrix:
+script:
+  - python setup.py test
+
+stages:
+  - lint
+  - test
+
+jobs:
   include:
-    - python: "2.7"
-    - python: "3.3"
+    # Linting
+    - stage: lint
+      name: "Check linting with flake8"
+      install: pip install -r requirements.dev.txt
+      script: find . -name \*.py -exec flake8 {} +
```

Comment:
Could this be simplified as:

    script: flake8 pymarc

Reply:
we also need to lint the test module, and maybe other python files in the project's root (like the apply_licence_header proposed by dan), but I agree this was too complex, I've committed this, would it be better ?

```yml
    - stage: lint
      name: "Check linting with flake8"
      install: pip install -r requirements.dev.txt
      script: flake8 .
    - stage: lint
      name: "Check format with black"
      install: pip install -r requirements.dev.txt
      script: black --quiet --diff .
```